CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+- Upgraded postgresql to bitnami/postgresql:15.5.11
```

Comment:
can you add a bold comment here that this is an upgrade of postgres and requires an upgrade of the database, which can be done using a dump and restore.

Reply:
Updated the content to "Upgraded postgresql to bitnami/postgresql:15.5.11. This requires an upgrade of the database, which can be done using a dump and restore."